Used load_dotenv, bot reponds when mentioned

SinghalChinmay opened the pull request:
I have made some changes to the bot.py file, now the bot responds when mentioned, as well as when we use its prefix "." . Also I used "load_dotenv" instead of "dotenv_values", which in my opinion is easier to use rather than "dotenv_values". 
A screenshot of the bot -
![Screen shot](https://cdn.discordapp.com/attachments/884651464000147476/894920449232867358/unknown.png)

Also not to mention, when using "load_dotenv", your file should be named ".env" and not in ".env.sample"

mogiiee commented:
@SinghalChinmay The first feature is seems useful but could you please revert the change of using dotenv itself.

SinghalChinmay commented:
@mogiiee 
I have fixed the things u said, also I fixed the quote command, cuz it didn't seem to work, I used aiohttp to get the quote as using requests module in async/await programming can cause issues

mogiiee commented:
@SinghalChinmay  This PR has merge conflicts with the main branch please resolve them for further action.
